src/settings-ui/Settings.UI/SettingsXAML/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -27,16 +27,17 @@
         <TextBlock
             x:Name="Header"
             MaxWidth="{StaticResource PageMaxWidth}"
-            Padding="36,0"
+            Padding="24,0"
```

Comment:
@jay-o-way What is the advantage of all of these changes here? Seems the XAML is becoming more complex but not visual changes are intended?

Reply:
Not sure if this answers your question, but I guess my brain has a different approach for implementing these things?

Reply:
I just want to make sure we are not breaking things that doesn't need fixing :)! If you can verify that there are no regressions (on different window sizes) this is fine!